### DISCUSSION TITLE
Sign and send transaction react native

### DISCUSSION THREAD

**Discussion by Badra00** [WalletConnect/walletconnect-monorepo#3413] [Developer Support] [answered]
Hello
I'm trying to sign and send a transaction on react native, I looked to the example but when trying to use the same code look like it's not updated or the ts definition are wrong since I don't find some methods.
Could you give me an example?

**Comment by glitch-txs** [comment, marked as answer]
hey have you looked through https://github.com/WalletConnect/modal-react-native/tree/main/example

**Comment by Badra00** [comment]
Yes I have looked, I manage to move further the version of ethers was has some breaking changes since your example, so I took the same. But now when sending the transaction nothing happen do you have any idea why?

**Reply by glitch-txs** [reply]
hi, is your code public? could you share a minimal example to reproduce this issue?

**Reply by Badra00** [reply]
Hey
I found the issue, I was not using the good object when using the signer now it works
Thanks
